packages/ui5-app/package.json

```diff
@@ -32,12 +33,20 @@
   },
   "devDependencies": {
     "@openui5/ts-types": "^1.60.5",
```

Comment:
`@openui5/ts-types` and `@sapui5/ts-types` - only one should be in - I would suppose we use the latest @sapui5 one

Reply:
done